src/index.js

```diff
@@ -5,11 +5,20 @@ import {
   curdFunctionality,
   retrieveTodoListFromLocalStorage,
 } from './curd.js';
+import { clearCompleted } from './status.js';
 
 document.addEventListener('DOMContentLoaded', () => {
+  curdFunctionality();
+
   const storedTodoList = retrieveTodoListFromLocalStorage();
   todoList.push(...storedTodoList);
 
-  curdFunctionality();
+  const clearButton = document.querySelector('.clearAll');
+  clearButton.addEventListener('click', (e) => {
+    e.preventDefault();
+    clearCompleted(todoList);
+  });
+  populateList();
 });
-document.addEventListener('DOMContentLoaded', populateList);
\ No newline at end of file
+
+// document.addEventListener('DOMContentLoaded', populateList);
```

Comment:
- [ ] @umarahgul Its not good practise having commented code in your codebase, kindly consider removing this commented code.